idea_town/experiments/tests.py

```diff
@@ -10,20 +13,120 @@
+                        "url": "http://testserver/api/experiments/%s" %
+                               experiment.pk,
+                        "title": experiment.title,
+                        "slug": experiment.slug,
+                        "thumbnail": None,
+                        "description": experiment.description,
+                        "measurements": experiment.measurements.rendered,
+                        "version": experiment.version,
+                        "changelog_url": experiment.changelog_url,
+                        "contribute_url": experiment.contribute_url,
+                        "created": date_field.to_representation(
+                            experiment.created),
+                        "modified": date_field.to_representation(
+                            experiment.modified),
+                        "xpi_url": "",
+                        "details": [],
+                        "contributors": []
+                    } for (slug, experiment) in self.experiments.items()),
+                    key=lambda x: x['id'])
+            }
+        )
+
+    def test_contributors(self):
+        """Experiment detail API resource should list contributor profiles"""
+        user_1 = self.users['experimenttest-1']
+        user_2 = self.users['experimenttest-2']
+
+        profile_1 = UserProfile.objects.get_profile(user_1)
+        profile_1.title = 'chief cat wrangler'
+        profile_1.display_name = 'jane doe'
+        profile_1.save()
+
+        profile_2 = UserProfile.objects.get_profile(user_2)
+        profile_2.title = 'assistant cat wrangler'
+        profile_2.display_name = 'john doe'
+        profile_2.save()
+
+        experiment = self.experiments['test-1']
+        experiment.contributors.add(user_1)
+        experiment.contributors.add(user_2)
+
+        url = reverse('experiment-detail', args=(experiment.pk,))
+        resp = self.client.get(url)
+        # HACK: Use a rest framework field to format dates as expected
+        date_field = fields.DateTimeField()
+        self.assertJSONEqual(
+            str(resp.content, encoding='utf8'),
+            {
+                "id": experiment.pk,
+                "url": "http://testserver/api/experiments/%s" %
+                       experiment.pk,
+                "title": experiment.title,
+                "slug": experiment.slug,
+                "thumbnail": None,
+                "description": experiment.description,
+                "measurements": experiment.measurements.rendered,
+                "version": experiment.version,
+                "changelog_url": experiment.changelog_url,
+                "contribute_url": experiment.contribute_url,
+                "xpi_url": "",
+                "details": [],
+                "created": date_field.to_representation(
+                    experiment.created),
+                "modified": date_field.to_representation(
+                    experiment.modified),
+                "contributors": [
```

Comment:
It would probably be cleaner to _just_ have this test verify that the contributor data is displayed--otherwise breaking the experiment API in other ways will produce a false failure here.